scipy/linalg/tests/test_special_matrices.py

```diff
@@ -278,6 +279,78 @@ def test_basic(self):
         assert_array_equal(a, expected)
 
 
+class TestHelmert(TestCase):
+
+    def test_orthogonality(self):
+        for n in range(1, 7):
+            H = helmert(n)
+            I = np.eye(n)
+            assert_allclose(H.dot(H.T), I, atol=1e-12)
+            assert_allclose(H.T.dot(H), I, atol=1e-12)
+
+    def test_subspace(self):
+        for n in range(2, 7):
+            U = helmert(n)[1:, :].T
+            C = np.eye(n) - np.ones((n, n)) / n
+            assert_allclose(U.dot(U.T), C)
+            assert_allclose(U.T.dot(U), np.eye(n-1), atol=1e-12)
+
+    def test_aitchison_geometry(self):
+
+        def _closure(w):
+            return w / w.sum()
```

Comment:
Yes, this is jargon from Aitchison geometry, along with 'perturbation' and 'powering'.  If the unfamiliar notation in this test is causing more friction than it's worth, maybe the whole test could just be deleted.